### ISSUE TITLE
Implementation of Entropic Open-Set Loss

### ISSUE THREAD

**Issue by ThiliniDahanayaka** [Vastlab/Reducing-Network-Agnostophobia#2]
In MNIST/Mnist_training.py, the 'categorical_crossentropy' is used in place of Entropic Open-Set loss. Is this accurate or should we implement custom implementation for the  Entropic Open-Set loss as defined in the paper?

**Comment by akshay-raj-dhamija** [comment]
Yes, you are right we use categorical_crossentropy rather than a custom implementation of Entropic Open-Set loss. This is possible because if you note the training data being provided for unknowns, rather than containing a one hot vector as in normal classification tasks contains a vector where all elements are of equal value that sum up to one. https://github.com/Vastlab/Reducing-Network-Agnostophobia/blob/581b3f06aeedb737e83a8610573f02299e2416b2/MNIST/Mnist_Training.py#L158

Please note, if you are using a setup other than keras+tensorflow, this may/may not work in which case it is better to create your custom loss function.

**Comment by dahanayaka** [comment]
Thank you so much for the clarification.

**Comment by ThiliniDahanayaka** [comment]
Thank you so much for the clarification.
